src/Mimi.jl

```diff
@@ -137,6 +153,8 @@ type Model
         m.parameters_that_are_set = Set{UTF8String}()
         m.parameters = Dict{Symbol, Parameter}()
         m.numberType = numberType
+        m.edges = []
```

Comment:
Does this work? I would think ``[]`` creates an array of type ``Array{Any,1}``. ``Array(ParameterVariableConnection, 0)`` Should create a vector with the right types and 0 elements.